ndcube/global_coords/global_coords.py

```diff
@@ -28,6 +30,20 @@ def _all_coords(self):
         A dynamic dictionary of all global coordinates, stored here or derived
         from the ndcube object.
         """
+        if self._ndcube is None:
+            return self._internal_coords
+
+        if hasattr(self._ndcube.wcs.low_level_wcs, "dropped_world_dimensions"):
+            dropped_world = self._ndcube.wcs.low_level_wcs.dropped_world_dimensions
+            extra_internal = {}
+            if dropped_world != {}:
+                for i in range(len(dropped_world["value"])):
+                    name = "dropped_world_axis_{}".format(i)
```

Comment:
I think given all the other places we use names and stuff, I think the best thing to do here would be to try the name key and fallback to the physical type.
```suggestion
                    name = dropped_world["world_axis_names"][i] or dropped_world["world_axis_physical_types"][i]
                    
```

Reply:
I didn't realize this syntax worked for empty strings as well as `None`, good to know. Made the changes suggested and updated the tests accordingly.